hops/hdfs.py

```diff
@@ -737,5 +741,7 @@ def localize(hdfs_path):
         Return an absolute path for local file/directory.
     """
 
-    return copy_to_local(hdfs_path, "", overwrite=True)
+    import os
```

Comment:
This line can be removed, os is already imported for the module?

Reply:
Weird. The new code is this instead of line 740. Looks ok in my branch, and my branch is up to date (pushed latest changes):
    import os
    filename = os.path.basename(hdfs_path)
    return copy_to_local(hdfs_path, filename)